src/controllers/controller.line.js

```diff
@@ -29,9 +26,14 @@ defaults._set('line', {
 	}
 });
 
-function lineEnabled(dataset, options) {
-	return valueOrDefault(dataset.showLine, options.showLines);
-}
+defaults._set('global', {
```

Comment:
Per our Slack exchange, this should finally be `defaults._set('datasets', { line: ... }` since we don't want it merged in the user options.

Reply:
Done